eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -140,6 +133,10 @@ jobs:
         parameters:
           SourceDirectory: $(Build.SourcesDirectory)
 
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
```

Comment:
Lets add this check to the aggregate pipeline as well and have that verify all md files in the repo.